Update access embedded rancher & longhorn (#269)
harvester/docs: docs/troubleshooting/harvester.md

- Enter a useful description for the support bundle and click `Create` to generate and download a support bundle.
    ![](/img/v1.1/troubleshooting/harvester-sb-support-modal.png)

> ## Access Embedded Rancher and Longhorn Dashboard

Reviewer: ```suggestion
## Access Embedded Rancher and Longhorn Dashboards
```

_Available as of v1.1.0_

> You can now access the embedded Rancher and Longhorn dashboard directly on the `Support` page, but you must enable the `Developer Tools & Features` first via the Preference page.

Reviewer: ```suggestion
You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must enable the `Developer Tools & Features` first via the Preference page.
```

![](/img/v1.1/troubleshooting/support-access-embedded-ui.png)

For previous versions, you can access them manually through:
- `https://{{HARVESTER_IP}}/dashboard/c/local/explorer` (Embedded Rancher)
- `https://{{HARVESTER_IP}}/dashboard/c/local/longhorn` (Embedded Longhorn)

:::note

> We only support to use the embedded Rancher and Longhorn dashboard for debugging and validation purpose.

Reviewer: ```suggestion
We only support using the embedded Rancher and Longhorn dashboards for debugging and validation purposes.
```
